core/build.sh

```diff
@@ -102,7 +102,7 @@ builder_describe_outputs \
   configure:wasm     build/wasm/$CONFIGURATION/build.ninja \
   build:x86          build/x86/$CONFIGURATION/src/libkmnkbp0.a \
   build:x64          build/x64/$CONFIGURATION/src/libkmnkbp0.a \
-  build:arch         build/arch/$CONFIGURATION/src/libkmnkbp0.a \
+  build:arch         build/arch/$CONFIGURATION/src/libkmnkbp0-static.a \
```

Comment:
The dynamic library is `libkmnkbp0.so`. I don't know if this change is valid for Mac?

Reply:
We are about to split the linux and mac builds here so we can resolve this separately.

Reply:
I think I will aim to produce both static and dynamic libraries for Core for all architectures. We already do this for Windows builds, and that will resolve this inconsistency everywhere.